include/OsmAndCore/Utilities.h

```diff
@@ -1221,10 +1221,17 @@ namespace OsmAnd
                     .arg(name)
                     .arg(color)));
         }
+        
+        inline static bool contains(const QVector<PointI> &polygon, const PointI &point) {
```

Comment:
We already have similar [method](https://github.com/osmandapp/OsmAnd-core/blob/master/src/Map/MapRasterizer_P.cpp#L818) Move it to Utilities if needed